### PR TITLE
[OpenShift] Adds e2e test for TektonAddon

### PR DESCRIPTION
**Pull request by sm43**
Signed-off-by: Shivam Mukhade <smukhade@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

**Comment by nikhil-thomas** [comment]
@pradeepitm12 @savitaashture

**Comment by nikhil-thomas** [comment]
LGTM

File: test/e2e/common/tektonconfigdeployment_test.go [hunk 1]
```diff
@@ -291,6 +291,55 @@ func runAddonTest(t *testing.T, clients *utils.Clients, tc *v1alpha1.TektonConfi
...
+	t.Run("validate-addon-params", func(t *testing.T) {
+
+		addonsIS, err := clients.Operator.TektonInstallerSets().List(context.TODO(), metav1.ListOptions{
+			LabelSelector: "operator.tekton.dev/created-by=TektonAddon",
+		})
+		if err != nil {
+			t.Fatalf("failed to get InstallerSet: %v", err)
+		}
+
+		// there must be 4 installerSet created for addons
```
Comment (savitaashture) [inline test/e2e/common/tektonconfigdeployment_test.go:304]: based on code it should be 5 :thinking: 
```suggestion
		// there must be 5 installerSet created for addons
```

Comment (savitaashture) [reply test/e2e/common/tektonconfigdeployment_test.go:304]: @sm43 its good to merge :+1: just above comment fix

**Review by pradeepitm12 — COMMENTED** [review]
LGTM

**Review by savitaashture — COMMENTED** [review]
:tada: 

/lgtm

**Pull request merged**